docs/en/rules/Azure.RedisEnterprise.MinTLS.md

```diff
@@ -0,0 +1,43 @@
+---
+severity: Critical
+pillar: Security
+category: Data protection
+resource: Azure Cache for Redis Enterprise
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Redis.MinTLS/
```

Comment:
@simone-bennett I believe the issue here with the path of the documentation. It is currently set to `Azure.Redis.MinTLS` when it should be set to `Azure.RedisEnterprise.MinTLS`